app/controllers/users_controller.rb

```diff
@@ -23,18 +25,21 @@ def edit
   # POST /users or /users.json
   def create
     @user = User.new(user_params)
+    @user.avatar.attach(user_params[:avatar])
 
-      if @user.save
-        flash[:success] = "Sign Up Process Successfull.Welcome #{@user[:name]} to the QI Blog.Please Login to Continue"
-        redirect_to login_url
-      else
-        flash[:success] = "Oops something went wrong!!"
-        render 'new'
-      end
+    if @user.save
+      flash[:success] = "Sign Up Process Successfull.Welcome #{@user[:name]} to the QI Blog.Please Login to Continue"
+      redirect_to login_url
+    else
+      flash[:danger] = "Oops something went wrong"
+      render 'new'
+    end
   end
 
   # PATCH/PUT /users/1 or /users/1.json
   def update
+    # @user.avatar.attach(user_params[:avatar])
```

Comment:
@HrisikeshSinghaChowdhury You need to replace the avatar with the latest image coming from params

Reply:
@HrisikeshSinghaChowdhury why this line is commented out?